sklearn/svm/base.py

```diff
@@ -538,7 +538,7 @@ def decision_function(self, X):
         dec = self._decision_function(X)
         if self.decision_function_shape is None and len(self.classes_) > 2:
             warnings.warn("The decision_function_shape default value will "
-                          "change from 'ovo' to 'ovr' in 0.18. This will change "
+                          "change from 'ovo' to 'ovr' in 0.19. This will change "
```

Comment:
Yes. Refer this - https://github.com/scikit-learn/scikit-learn/pull/5528/files#diff-51431c46fed3ff4601d253ebc9674fe3L615